template/Makefile.j2

```diff
@@ -59,7 +59,18 @@ docker-publish:
 	fi;\
 	# This generates a signature and publishes it to the registry, next to the image\
 	# Uses the keyless signing flow with Github Actions as identity provider\
-	cosign sign -y ${OCI_REGISTRY_HOSTNAME}/${OCI_REGISTRY_PROJECT_IMAGES}/${OPERATOR_NAME}:@$$REPO_DIGEST_OF_IMAGE
+	cosign sign -y ${OCI_REGISTRY_HOSTNAME}/${OCI_REGISTRY_PROJECT_IMAGES}/${OPERATOR_NAME}@$$REPO_DIGEST_OF_IMAGE;\
+	# Generate the SBOM for the operator image, this leverages the already generated SBOM for the operator binary by cargo-cyclonedx\
+	syft scan -o cyclonedx-json --exclude "/usr/local/bin/stackable-${OPERATOR_NAME}" --scope all-layers --source-name "${OPERATOR_NAME}" --source-version "${VERSION}" ${OCI_REGISTRY_HOSTNAME}/${OCI_REGISTRY_PROJECT_IMAGES}/${OPERATOR_NAME}@$$REPO_DIGEST_OF_IMAGE > sbom.json;\
```

Comment:
Sorry for the naive question but....where does the semicolon belong to at the end of the lines?

Reply:
These commands are basically "one line of bash" but split into several lines (concatenated with `\`, which escapes the newline), hence they need a semicolon between them (could also use `&&` I think). The intention was to be able to write `PURL=...` instead of `export PURL=...`. I should probably do it with `export` though, is prettier now that there are several lines involved. I'll change it.

Reply:
Can/should we maybe even factor all of this out into a script that you just call here?

Reply:
There are `make` variables involved, which we would probably need to pass as arguments to the script. So I think it would be possible, but maybe we should create a separate issue for this. It's quite cumbersome to write these Makefiles anyway, since there are a lot of abstractions: I edit the Jinja2 template, a Makefile gets rendered which includes shell script which is executed by Github Actions (which brings in some variables/secrets as environment variables). This sometimes results in difficult escaping of symbols and is hard to test locally, maybe we could find a better solution for our build process (better reproducibility in general would also be nice).

Reply:
Update on the `export` approach: I could not `make` it work :disappointed: Had problems with variables being out of scope / undefined. I'll leave it for now.